<?xml version='1.1' encoding='UTF-8'?>
<Tag plugin="workflow-support@3.8">
  <node class="cps.n.StepAtomNode" plugin="workflow-cps@2.94">
    <parentIds>
      <string>38</string>
    </parentIds>
    <id>39</id>
    <descriptorId>org.jenkinsci.plugins.workflow.steps.durable_task.ShellStep</descriptorId>
  </node>
  <actions>
    <cps.a.ArgumentsActionImpl plugin="workflow-cps@2.94">
      <arguments>
        <entry>
          <string>returnStdout</string>
          <boolean>true</boolean>
        </entry>
        <entry>
          <string>script</string>
          <string>curl -s -w &apos;
%{response_code}&apos; -X POST --header &apos;accept: application/json&apos; --header &apos;x-api-key: Njc4OTA5MGVlY2QyNDhlYTBjY2MzN2E4MjgzNmRmMjI4ZWM4OWFiZDE1ZDY1OWUyZWU2MmFhOGNkMjBiNDVlYTQ5&apos; --header &apos;Content-Type: multipart/form-data&apos; --form &apos;file[]=@&quot;&quot;&apos; --form &apos;os=&quot;windows&quot;&apos; --form &apos;platform=&quot;web&quot;&apos; --form &apos;build=&quot;50&quot;&apos; --form &apos;envelope=&quot;false&quot;&apos; --form &apos;smart=&quot;true&quot;&apos; https://app.calliope.pro/api/v2/profile/3792/import/testng -v</string>
        </entry>
      </arguments>
      <sensitiveVariables/>
      <isUnmodifiedBySanitization>true</isUnmodifiedBySanitization>
    </cps.a.ArgumentsActionImpl>
    <wf.a.TimingAction plugin="workflow-api@2.47">
      <startTime>1636033919792</startTime>
    </wf.a.TimingAction>
    <s.a.LogStorageAction/>
    <wf.a.ErrorAction plugin="workflow-api@2.47">
      <error class="hudson.AbortException">
        <detailMessage>script returned exit code 26</detailMessage>
        <stackTrace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.handleExit(DurableTaskStep.java:659)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.check(DurableTaskStep.java:605)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.run(DurableTaskStep.java:549)</trace>
          <trace>java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)</trace>
          <trace>java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)</trace>
          <trace>java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)</trace>
          <trace>java.base/java.lang.Thread.run(Thread.java:829)</trace>
        </stackTrace>
        <suppressedExceptions class="empty-list"/>
      </error>
    </wf.a.ErrorAction>
  </actions>
</Tag>